infra/split.dockerfile

```
@@ -105,6 +105,15 @@ RUN --mount=type=cache,target=/root/.local/share/pnpm/store pnpm --filter=docs..
 COPY . .
 RUN --mount=type=cache,target=/app/.turbo pnpm run build --filter=docs
 
+# Builder for Admin
+FROM base-builder AS admin-builder
+COPY .npmrc package.json pnpm-lock.yaml pnpm-workspace.yaml ./
+COPY --parents packages/**/package.json .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)